src/backend/utils/misc/etc/template-hawq-site.xml

```diff
@@ -20,131 +20,133 @@ under the License.
 -->
 
 <configuration>
-    <property>
-        <name>hawq_master_address_host</name>
-        <value>%master.host%</value>
-    </property>
-
-    <property>
-        <name>hawq_master_address_port</name>
-        <value>%master.port%</value>
-    </property>
-
-    <property>
-        <name>hawq_standby_address_host</name>
-        <value>%standby.host%</value>
-    </property>
-
-    <property>
-        <name>hawq_segment_address_port</name>
-        <value>%segment.port%</value>
-    </property>
-
-    <property>
-        <name>hawq_dfs_url</name>
-        <value>%namenode.host%:%namenode.port%/%hawq.file.space%</value>
-    </property>
-
-    <property>
-        <name>hawq_master_directory</name>
-        <value>%master.directory%</value>
-    </property>
-
-    <property>
-        <name>hawq_segment_directory</name>
-        <value>%segment.directory%</value>
-    </property> 
-
-    <property>
-        <name>hawq_master_temp_directory</name>
-        <value>%master.temp.directory%</value>
-    </property>
-
-    <property>
-        <name>hawq_segment_temp_directory</name>
-        <value>%segment.temp.directory%</value>
-    </property>
+	<property>
+		<name>hawq_master_address_host</name>
+		<value>%master.host%</value>
+		<description>The host name of hawq master.</description>
+	</property>
+
+	<property>
+		<name>hawq_master_address_port</name>
+		<value>%master.port%</value>
+		<description>The port of hawq master.</description>
+	</property>
+
+	<property>
+		<name>hawq_standby_address_host</name>
+		<value>%standby.host%</value>
+		<description>The host name of hawq standby master.</description>
+	</property>
+
+	<property>
+		<name>hawq_segment_address_port</name>
+		<value>%segment.port%</value>
+		<description>The port of hawq segment.</description>
+	</property>
+
+	<property>
+		<name>hawq_dfs_url</name>
+		<value>%namenode.host%:%namenode.port%/%hawq.file.space%</value>
+		<description>URL for accessing HDFS.</description>
+	</property>
+
+	<property>
+		<name>hawq_master_directory</name>
+		<value>%master.directory%</value>
+		<description>The directory of hawq master.</description>
+	</property>
+
+	<property>
+		<name>hawq_segment_directory</name>
+		<value>%segment.directory%</value>
+		<description>The directory of hawq segment.</description>
+	</property> 
+
+	<property>
+		<name>hawq_master_temp_directory</name>
+		<value>%master.temp.directory%</value>
+		<description>The temporary directory reserved for hawq master.</description>
+	</property>
+
+	<property>
+		<name>hawq_segment_temp_directory</name>
+		<value>%segment.temp.directory%</value>
+		<description>The temporary directory reserved for hawq segment.</description>
+	</property>
      
-    <!-- HAWQ resource manager parameters -->
-    <property>
-        <name>hawq_rm_yarn_address</name>
-        <value>%master.host%:9980</value>
```

Comment:
To make the ports consistent in hawq-site.xml, template-hawq-site.xml and yarn-client.xml, will use below settings:

hawq_rm_yarn_address: %master.host%:8032
hawq_rm_yarn_scheduler_address: %master.host%:8030